app/scripts/modules/kubernetes/serverGroup/configure/wizard/wizard.html

```diff
@@ -5,7 +5,7 @@
   <div ng-if="!state.loaded" style="height: 200px">
     <h3 us-spinner="{radius:30, width:8, length: 16}"></h3>
   </div>
-  <v2-modal-wizard ng-show="state.loaded && !state.requiresTemplateSelection" heading="{{title}}">
+  <v2-modal-wizard ng-if="state.loaded && !state.requiresTemplateSelection" heading="{{title}}">
```

Comment:
ng-show switches a tag's CSS ("display: none" / "display: visible"), but the template will render. A tag with ng-if="false" won't render at all.